portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,173 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script>$('.carousel').carousel({
+      interval: 20
+    })</script>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+
+  <body onload="addRandomQuote()">
+    <div id="nav-placeholder"></div>
+
+    <nav class="navbar  fixed-top navbar-expand-lg navbar-dark bg-dark">
+      <a class="navbar-brand" href="#">Jose's Portfolio</a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+            <a class="nav-link" href="#hobbies">Hobbies</a>
+          </li>
+          <li class="nav-item active">
+            <a class="nav-link" href="#code">Code</a>
+          </li>
+          <li class="nav-item active">
+            <a class="nav-link" href="#places">Places</a>
+          </li>
+      </div>
+    </nav>
+
+    <br>
+    <br>
+
+    <div class="jumbotron text-center container-fluied">
+      <h2 >Here is a quote that I like:</h2>
+      <blockquote class="blockquote">
+        <p id ="quote-container" class="mb-0 "></p>
+        <footer id="name-container" class="blockquote-footer"><cite title="Source Title"></cite></footer>
+      </blockquote>
     </div>
+    <div id="hobbies" class="section1 grad1 container-fluid">
+    <h1 style="color:#000000">Hobbies</h1>
+
+    <div class="card-group">
+      <div class="card">
+        <img src="images/food.jpg" class="cropped card-img-top" alt="...">
```

Comment:
`alt=".."` text isn't populated -- ideally this would be a description of the image (for e.g. text browsers/screen readers), or just left off if not needed.